ruby/spec/find_cache_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'find_cache'
 
 RSpec.describe FindCache do
-  let(:bucket) { 'outstand-buildkite-cache' }
+  let(:bucket) { 'outstand-buildkite-data' }
```

Comment:
I changed the bucket for these specs to a newer one that doesn't expire the keys. This means the specs should keep working longer than a couple weeks!